common/app/views/fragments/header.scala.html

```diff
@@ -67,21 +67,21 @@
                     @if(metaData.section == "observer" && metaData.isFront) {
                         <a href="@LinkTo{@metaData.url}" data-link-name="site logo" id="logo" class="logo-wrapper logo-wrapper--observer" itemprop="url" data-component="logo">
                             @if(metaData.hasSlimHeader) {
-                                <i class="i i-observer-logo-160"></i>
+                                @fragments.inlineSvg("observer-logo-160", "logo")
                             } else {
                                 <span class="u-h" itemprop="name">The Observer</span>
-                                <i class="i i-observer-logo-160 mobile-only"></i>
-                                <i class="i i-observer-logo-320 hide-on-mobile"></i>
+                                @fragments.inlineSvg("observer-logo-160", "logo", List("mobile-only"))
+                                @fragments.inlineSvg("observer-logo-320", "logo", List("hide-on-mobile"))
                             }
                         </a>
                     } else {
                         <a href="@LinkTo{/}" data-link-name="site logo" id="logo" class="logo-wrapper" itemprop="url" data-component="logo">
                             <span class="u-h" itemprop="name">The Guardian</span>
                             @if(metaData.hasSlimHeader) {
-                                <i class="i i-guardian-logo-160"></i>
+                                @fragments.inlineSvg("guardian-logo-160", "logo")
```

Comment:
@marchibbins This does affect the identity header as it has its own template and CSS file. I'd prefer to leave and then do the work to replace the identity header with the new slim nav.